airflow/__init__.py

```diff
@@ -15,13 +15,15 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from __future__ import annotations
 
 # Make `airflow` a namespace package, supporting installing
 # airflow.providers.* in different locations (i.e. one in site, and one in user
 # lib.)  This is required by some IDEs to resolve the import paths.
+# And it _has_ to be the first code line too. Sad panda
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
 
+from __future__ import annotations  # noqa: F404
```

Comment:
Although:

>Python mandates that future-imports must appear in the module before any other code except docstrings

https://peps.python.org/pep-0008/#module-level-dunder-names

Reply:
Question to the audience - can we live with it for now until we find a better alternative?

Reply:
Yeah, for now I think we can live with that bug.

Once we move a good number of things from Airflow to SDK, maybe the `airflow/__init__.py` file won't have much content left to keep `from __future__ import annotations`!